### DISCUSSION TITLE
first public release

### DISCUSSION THREAD

**Discussion by masqu3rad3** [masqu3rad3/tik_manager4#41] [Announcements]
🔧 What to expect:
No easy installer yet, but it's a breeze to install! If you're unsure, just DM me for guidance.
No documentation at this stage, but I'm here to help! Feel free to reach out with any questions.
Remember, it's an alpha, so use it for testing purposes only. Expect changes until the beta stage.
📚 Stay tuned: More features, bug fixes, and workflows will be documented in the next official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/masqu3rad3/tik_manager4/releases/tag/v4.0.1-alpha'>first public release</a>.</em>

**Comment by bodibod** [comment]
Hey Arda, how are you doing ?
Could you make a quick how to install  (linux) ? Would be dope

**Reply by masqu3rad3** [reply]
There is an online documentation which will apply to the next release https://github.com/masqu3rad3/tik_manager4/releases/tag/v4.0.2-beta

The online docs are here:
https://tik-manager4.readthedocs.io/en/latest/

**Comment by ritwik-github** [comment]
Hey Arda, I am so excited to test your tool .. and one request can you please give it a try for Katana (foundry), mari (foundry) please?

**Reply by masqu3rad3** [reply]
Hi, I will try both. Some dccs are super easy to integrate and some are very tricky. Mari and Katana may be the tricky ones at least for me since I am not familiar with their workflow. Any help on that would be appreciated :)

**Reply by ritwik-github** [reply]
yeah true... I am a regular user of Katana though ... will be very happy to help
